notebooks/official/automl/automl_forecasting_bqml_arima_plus_comparison.ipynb

```diff
@@ -0,0 +1,1689 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    REGION = "us-central1"  # @param {type: "string"}
</pre><p>Please inject this according to https://docs.google.com/document/d/1MjbMDrm4GvfLJfqnPmdTUS8dueKdRAUsn5bmm2aCoZo/edit?resourcekey=0-LE_XfwNqTclOKG8hbFnxrQ#bookmark=id.61pzrvibw4mf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/581/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>